CIP-0027.md

```diff
@@ -0,0 +1,76 @@
+## Simple Summary
+
+Create a standard tagging for royalties&#39; distribution within the metadata of Cardano Assets.
+
+## Abstract
+
+This proposed tagging standard will allow for uniform royalties&#39; distributions. It has been developed with input from Artano, CNFT.io, Digital Syndicate, and Buffy Bot.
+
+## Motivation
+
+There is a significant number of artists who have expressed desire to leverage a royalties function within Cardano Assets, which will allow them a percentage of the future resale of assets they have created. There is also a misperception that Smart Contracts will have a built in solution for royalties. As of the writing of this CIPS proposal, Smart Contracts do not have this capability. By creating a standard for metadata tags, it will allow for immediate adoption, and backwards compatibility.
+
+## Specification
+
+&quot;royalties&quot; - Placed within CNFT Metadata.
+ Can either be an array that contains the royalties instructions or contains a link to offchain royalties json which includes the royalties instructions. The link to an offchain json is recommended so that it can be modified in the future.
+
+Royalties instructions tags:
+
+&quot;royaltiesContact&quot;: contact info for royalties inquiries (optional)
+ &quot;royaltiesInstructionVersion&quot;: version of formatting used
+ &quot;royaltiesPercent&quot;: Overall percentage of resale to be distributed as royalties
+ &quot;royaltiesPrimaryOwner&quot;: Friendly name of first royalties party (optional)
```

Comment:
Probably this should be an array instead of duplicating key names

Reply:
The logic here is that a marketplace would read in the entire metadata for a given NFT, which would include the offchain data.  For that reason, we wouldnt want to have any potential overlapping tags between the onchain and offchain metadata.  By starting each tag with "royalties", we silo this component off.  However, its certainly open for discussion.  And obviously the goal is to find a happy medium that all of the key stakeholders in the community can agree upon.